### DISCUSSION TITLE
(java sdk ) ssm - ParameterStringFilter

### DISCUSSION THREAD

**Discussion by litalhalawi** [aws/aws-sdk-java#2952] [Q&A] [answered]
hey all,
when using JAVA1 dependency:
```
<groupId>com.amazonaws</groupId>-->
<artifactId>aws-java-sdk</artifactId>-->
```
 I could add a filter for parameters with name starts with /company/site/param 
while using the new dependency:
```
<dependency>
            <groupId>software.amazon.awssdk</groupId>
            <artifactId>ssm</artifactId>
            <version>2.20.32</version>
        </dependency>
```

it fails.... and I di not get my results from store.
ParameterStringFilter.builder().key("Name").option("BeginsWith").values("/abc/abc/abc/as").build()

am I doing something wrong?

**Comment by ashishdhingra** [comment, marked as answer]
Closed by requestor.
